packages/node_modules/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -300,12 +300,39 @@ skipInBrowser(describe)('plugin-meetings', () => {
         });
       });
       describe('#create', () => {
+        let infoOptions;
+
         it('should have #create', () => {
           assert.exists(webex.meetings.create);
         });
         beforeEach(() => {
+          infoOptions = {
+            destination: 'dest-example',
+            type: 'type-example'
+          };
+
           webex.meetings.createMeeting = sinon.stub().returns(Promise.resolve({on: () => true}));
         });
+
+        it('should call MeetingInfo#fetchInfoOptions() with proper params',
```

Comment:
As long as it calls this method with the appropriate values, it will always proceed with the proper assignments at a later time based on previously implemented tests.